### ISSUE TITLE
ofxFontStash does not work with OpenGL 3.+ contexts

### ISSUE THREAD

**Issue by entd** [armadillu/ofxFontStash#23]
ofxFontStash completely does not work with OpenGL 3.+ contexts. While by default it uses old context, support of the new context is probably even more important, given that they are common, despite openframeworks defaults.

**Comment by armadillu** [comment]
Yep its on my list but I started work on a rewrite at https://github.com/armadillu/ofxFontStash2

**Comment by alptugan** [comment]
the same issue is relevant on oF 0.11. Cannot display font glyphs with glVersion(3,2)
